modules/redisearch/redisearch.ts

```diff
@@ -251,10 +251,28 @@ export class Redisearch extends Module {
      * @param options The additional optional parameters
      * @returns An array, in which each element represents a misspelled term from the query
      */
-    async spellcheck(index: string, query: string, options?: FTSpellCheck): Promise<string[]> {
+    async spellcheck(index: string, query: string, options?: FTSpellCheck): Promise<FTSpellCheckResponse[]> {
         const command = this.searchCommander.spellcheck(index, query, options);
         const response = await this.sendCommand(command);
-        return this.handleResponse(response);
+        return this.handleSpellcheckResponse(response);
+    }
+
+    handleSpellcheckResponse(response: any): FTSpellCheckResponse[] {
```

Comment:
Do you think this should be inside a 'redisearch.helpers.ts' file?

Reply:
I think it belongs here more than to the commander file.
But if you have any specific ideas, I would be happy to do it that way.

Reply:
I meant create a new file of helpers for redisearch. Not the commander.
Wdyt?